plugins/inventory/foreman.py

```diff
@@ -85,6 +85,64 @@
         description: Number of hosts per batch that will be retrieved from the Foreman API per individual call
         type: integer
         default: 250
+      foreman:
+        description: foreman server related configuration
+        type: dict
+        use_reports_api:
+            description: Use Reporting API.
+            type: boolean
+            default: False
+      report:
+        description: report API specific configuration
+        type: dict
+        poll_interval:
+            description: The polling interval between 2 calls to the report_data endpoint while polling.
+            type: integer
```

Comment:
these should be `type: int`, not that Ansible properly parses sub-options for inventories anyways.